tests/test_primitives.py

```diff
@@ -31,11 +32,13 @@ class TestModel(BaseXmlModel, tag='model'):
 
         element1: float = element()
         element2: dt.datetime = element()
+        element3: ipaddress.IPv4Address = element()
```

Comment:
I think it worth to write a separate test for `ipaddress` types serialisation and test all of them.